spec/components/checkboxes_spec.rb

```diff
@@ -88,6 +88,19 @@ def component_name
     assert_select ".govuk-hint", text: "Choose carefully"
   end
 
+  it "renders checkboxes with no hint" do
+    render_component(
+      name: "favourite_colour",
+      no_hint_text: true,
+      items: [
+        { label: "Red", value: "red" },
+        { label: "Green", value: "green" },
+      ]
+    )
+    assert_select ".govuk-hint", false
+    assert_select ".govuk-fieldset[aria-describedby]", false
```

Comment:
This should possibly be a separate test, but as long as there's a test covering this situation it probably doesn't matter too much!

Reply:
I think the two are linked... actually we should probably test from the other side as well i.e. that describedby is set correctly when hint text isn't hidden.